src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -702,7 +702,7 @@ internal MethodSymbol TryFindDisposePatternMethod(TypeSymbol exprType, SyntaxNod
         {
             LookupResult lookupResult = LookupResult.GetInstance();
             
-            MethodSymbol disposeMethod = FindPatternMethod(exprType, WellKnownMemberNames.DisposeMethodName, lookupResult, syntaxNode, warningsOnly: true, diagnostics, syntaxNode.SyntaxTree, MessageID.IDS_Disposable);
+            MethodSymbol disposeMethod = FindPatternMethod(exprType, WellKnownMemberNames.DisposeMethodName, lookupResult, syntaxNode, searchExtensionMethods: true, warningsOnly: true, diagnostics, syntaxNode.SyntaxTree, MessageID.IDS_Disposable);
```

Comment:
Could you reformat this like:

```FindPatternMethod(
    exprType, 
    WellKnownMemberNames.DisposeMethodName, 
    lookupResult, syntaxNode, searchExtensionMethods: true,
    warningsOnly: true,
    diagnostics,
    syntaxNode.SyntaxTree,
    MessageID.IDS_Disposable)
```

The line is so long right now that it's pretty unreadable.